lib/model/store.dart

```diff
@@ -570,7 +572,18 @@ class LiveGlobalStore extends GlobalStore {
 
   @override
   Future<Account> doInsertAccount(AccountsCompanion data) async {
-    final accountId = await _db.createAccount(data); // TODO(log): db errors
+    final int accountId;
+    try {
+      accountId = await _db.createAccount(data); // TODO(log): db errors
+    } catch (e) {
+      if (e case DriftRemoteException(
+            remoteCause: SqliteException(
+              extendedResultCode: SqlExtendedError.SQLITE_CONSTRAINT_UNIQUE))) {
+          throw AccountAlreadyExistsException();
```

Comment:
nit in commit message:

    store: Throw exception when account already logged in
    
    When account is already logged in, throw AccountAlreadyExists to allow caller
    to catch and handle exception.

Line too long in the prose; see https://zulip.readthedocs.io/en/latest/contributing/commit-discipline.html#formatting-guidelines .